synapse/tools/genpkg.py

```diff
@@ -224,7 +225,7 @@ def loadPkgProto(path, opticdir=None, no_docs=False, readonly=False):
     s_schemas.reqValidPkgdef(pkgdef)
 
     # Ensure the package is json safe and tuplify it.
-    s_common.reqJsonSafeStrict(pkgdef)
+    s_json.reqjsonsafe(pkgdef)
```

Comment:
per discussion - we'll need to make sure we have a `s_json.reqJsonSafeStrict` function available.

Reply:
I added an optional `strict` kwarg to `s_json.reqjsonsafe`